.gitlab/ci/global.yml

```diff
@@ -196,10 +196,10 @@
       - local: .gitlab/ci/slack-notify.yml
 
 
-.run-unittests-and-lint:
+.unittests-and-lint-settings:
```

Comment:
To reuse instead of duplicate code. For the new pipeline workflow, I need all the env preparations for running lint without running it as we always do. So I split it into settings and the run itself. Notice where these lines are located to see the split in the logic.